docs/GoogleAnalyticsStreamingSource-streamingsource.md

```diff
@@ -0,0 +1,33 @@
+# Google Analytics 360 streaming source
+
+Description
+-----------
+This plugin is used to query Google Analytics Reporting API.
+
+Using this plugin you can get metrics and dimensions with built-in set. 
+
+Please, see _https://developers.google.com/analytics/devguides/reporting/realtime/v3_ to get more information.
```

Comment:
Please add information about using limited beta api and project whitelisting.

Reply:
@CuriousVini added, please confirm